README.md

```diff
@@ -51,18 +51,29 @@ missing please open an issue. If you are interested in joining check out
 
 ### Infra Admins
 
-- [@gibfahn](https://github.com/gibfahn) - Gibson Fahnestock
+<!-- ncu-team-sync.team(nodejs/build-infra) -->
+
 - [@jbergstroem](https://github.com/jbergstroem) - Johan Bergström
 - [@joaocgreis](https://github.com/joaocgreis) - João Reis
 - [@mhdawson](https://github.com/mhdawson) - Michael Dawson
 - [@rvagg](https://github.com/rvagg) - Rod Vagg
 
+<!-- ncu-team-sync end -->
+
 ### Release Admins
 
-- [@gibfahn](https://github.com/gibfahn) - Gibson Fahnestock
+<!-- ncu-team-sync.team(nodejs/jenkins-release-admins) -->
```

Comment:
Not sure if this is the correct team (or if there is a team). This section is referenced from https://github.com/nodejs/build/blob/master/doc/access.md#release-servers and doesn't mention a GitHub team.

Reply:
I could be wrong, but I believe this GH team is people who can access ci-release.nodejs.org, which I do have access to, and my name is here.

That link is a bit odd, the description is

> The current list of Build WG members with infra access 

but then it links to here... infra is overused, sometimes it means the infra team, sometimes it means all "infrastructure" -- jenkins, build hosts, everything.

I'm not actually sure then, what this section heading refers to. It could also refer to those people who have ssh access to the release machines: https://github.com/nodejs-private/secrets/tree/master/build/release/.gpg

That list is surprisingly small, it looks like the (non admin) release team might use a shared ssh key to get into release hosts, I see "release@build.nodejs.org" in the authorized_keys for one of the hosts I have "special" access to in order to be able to ansible it.

@nodejs/build the people listed in this section, what do they have access to? Who should be here? Should jenkins-release-admins be a different section?

Reply:
`jenkins-release-admins` is referenced in https://github.com/nodejs/build/blob/master/doc/access.md#ci-releasenodejsorg and refers to the people who are allowed to admin the Jenkins instance on https://ci-release.nodejs.org/.